packages/bp/src/core/bots/bot-service.ts

```diff
@@ -266,10 +264,11 @@ export class BotService {
           .warn(`The bot ${botId} already exists, files in the archive will overwrite existing ones`)
       }
     }
-    const tmpDir = tmp.dirSync({ unsafeCleanup: true })
-    const tmpFolder = tmpDir.name
 
+    let tmpDir: tmp.SynchrounousResult | null = null
```

Comment:
Please use undefined whenever possible. Only place we use null is in the frontend if I remember correctly